ghcup-prereleases-0.0.8.yaml

```diff
@@ -1503,12 +1503,124 @@ ghcupDownloads:
         dlSubdir: ghc-9.10.0.20240313
         dlUri: https://downloads.haskell.org/~ghc/9.10.0.20240313/ghc-9.10.0.20240313-src.tar.xz
       viTags:
-      - LatestPrerelease
```

Comment:
This drops the whole prerelease tag.

It either has to be `Prerelease` or `LatestPrerelease`.